net_url/_urls.ts

```diff
@@ -0,0 +1,21 @@
+export class NetUrlUtils {
+    static HOMEPAGES = {
+        HTTPS: {
+            twitter: 'https://x.com',
+            quickbench: 'https://quick-bench.com',
+            cloogle: 'https://cloogle.org',
+            cppinsights: 'https://cppinsights.io',
+            static_ce_cdn: 'https://static.ce-cdn.net',
+            github: 'https://github.com',
+        },
+
+        HTTP: {
+            reddit: 'http://www.reddit.com',
+        },
```

Comment:
we already went over this in the other PR. we should instead just export each url as its own constant. there is no reason to introduce a class